src/rascal/structure_managers/adaptor_full_neighbour_list.hh

```diff
@@ -63,6 +63,7 @@ namespace rascal {
         typename LayerIncreaser<MaxOrder,
                                 typename parent_traits::LayerByOrder>::type{})};
     using LayerByOrder = std::index_sequence<AtomLayer, 0>;
+    typedef ManagerImplementation PreviousManager_t;
```

Comment:
I would prefer the `using` syntax compared to the typedef. There are other instances too that should be changed